webextensions/api/scripting.json

```diff
@@ -0,0 +1,399 @@
+              },
+              "edge": {
+                "version_added": "88"
+              },
+              "firefox": {
+                "version_added": "101"
+              },
+              "firefox_android": {
+                "version_added": "101"
+              },
+              "opera": {
+                "version_added": "74"
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            }
+          }
+        },
+        "executeScript": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/executeScript",
+            "support": {
+              "chrome": {
+                "version_added": "88"
+              },
+              "edge": {
+                "version_added": "88"
+              },
+              "firefox": {
+                "version_added": "101"
+              },
+              "firefox_android": {
+                "version_added": "101"
+              },
+              "opera": {
+                "version_added": "74"
+              },
+              "safari": {
+                "version_added": "15.4"
+              },
+              "safari_ios": {
+                "version_added": "15.4"
+              }
+            }
+          },
+          "injectImmediately": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "88"
+                },
+                "edge": {
+                  "version_added": "88"
+                },
+                "firefox": {
+                  "version_added": "101"
+                },
+                "firefox_android": {
+                  "version_added": "101"
+                },
+                "opera": {
+                  "version_added": "74"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              }
+            }
+          },
+          "world": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "88"
+                },
+                "edge": {
+                  "version_added": "88"
+                },
+                "firefox": {
+                  "version_added": "101",
+                  "notes": "Only supports <code>ISOLATED</code>, not <code>MAIN</code>."
+                },
+                "firefox_android": {
+                  "version_added": "101",
+                  "notes": "Only supports <code>ISOLATED</code>, not <code>MAIN</code>."
+                },
+                "opera": {
+                  "version_added": "74"
+                },
+                "safari": {
+                  "version_added": false
```

Comment:
Correction to what I wrote before: the `"world"` parameter is supported in Safari too, 15.4.

Reply:
@Rob--W  Safari and Safari iOS?